charts/ehrenamt-justiz/templates/serviceMonitor.yaml

```
@@ -0,0 +1,25 @@
+    - interval: {{ $module.serviceMonitor.interval | default "30s" }}
+      path: {{ $module.serviceMonitor.path | default "/actuator/metrics" }}
+      port: {{ $module.serviceMonitor.port | default "http" }}
+      scheme: {{ $module.serviceMonitor.scheme | default "http" }}
+  namespaceSelector:
+    matchNames:
+      - {{$.Release.namespace}}
+  selector:
+    matchLabels:
+      {{- include "getLabels" $data | trim | nindent 4 }}
+{{- end }}
+{{- end }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)